modules/check_in/app/sidekiq/check_in/travel_claim_status_check_worker.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+module CheckIn
+  class TravelClaimStatusCheckWorker < TravelClaimBaseWorker
+    def perform(uuid, appointment_date)
+      redis_client = TravelClaim::RedisClient.build
+      mobile_phone = redis_client.patient_cell_phone(uuid:)
+      station_number = redis_client.station_number(uuid:)
+      facility_type = redis_client.facility_type(uuid:)
+
+      logger.info({
+                    message: "Checking travel claim status for #{uuid}, #{appointment_date}, " \
+                             "#{station_number}, #{facility_type}",
+                    uuid:,
+                    appointment_date:,
+                    station_number:,
+                    facility_type:
+                  })
+
+      claim_number, template_id = claim_status(uuid:, appointment_date:, station_number:, facility_type:)
+
+      send_notification(mobile_phone:, appointment_date:, template_id:, claim_number:, facility_type:)
+      StatsD.increment(Constants::STATSD_NOTIFY_SUCCESS)
+    end
+
+    def claim_status(opts = {})
+      check_in_session = CheckIn::V2::Session.build(data: { uuid: opts[:uuid] })
+
+      claim_status_resp = TravelClaim::Service.build(
+        check_in: check_in_session,
+        params: { appointment_date: opts[:appointment_date] }
+      ).claim_status
+
+      handle_response(claim_status_resp:, facility_type: opts[:facility_type])
+    rescue => e
+      logger.error({ message: "Error calling BTSSS Service: #{e.message}" }.merge(opts))
```

Comment:
Should me make this log message specific so we can distinguish it from the submit claim log?

Reply:
Sure Gaurav, I will change it to "Error calling BTSSS/GetClaimsStatus API:"

Reply:
maybe the method can be included as a param, so:
```suggestion
      logger.error({ message: "Error calling BTSSS Service: #{e.message}", method: "claim_status" }.merge(opts))
```